src/actions/receive-profile.js

```diff
@@ -1431,7 +1431,8 @@ export function getProfilesFromRawUrl(
       case 'none':
       case 'from-file':
       case 'local':
-        throw new Error(`There is no profile to download`);
```

Comment:
<img width="765" alt="image" src="https://user-images.githubusercontent.com/1588648/92008757-2c312f00-ed0d-11ea-870b-09e50d9174fd.png">

Thank you! I was wanting to fix that.